docker/README.md

```diff
@@ -70,3 +70,26 @@ After build finished, run the container:
 ```
 docker run --rm -it lightgbm
 ```
+
+## Running the R-package Сontainer
+
+Build the container based on the [`verse` Rocker image](https://www.rocker-project.org/images/), for R users:
```

Comment:
Thanks! Why was it necessary to use the `verse` image instead of the lighter rocker images like `r-ver`? Similarly, could you explain the reasoning behind using an image with RStudio in it vs. one which can just run `LightGBM` in a vanilla R process?

I'm not opposed to the choices you've made, just want to understand the reasoning. I think if we leave these as-is, we might want to call this an "R training" image to note that we have some things (like RStudio) that are only necessary when doing interactive development.

Reply:
I see the Dockerfiles in this directory as a help for users who want to write their own `Dockerfile` with LightGBM to try it out. As there are no automated builds from the `Dockerfile`s (or at least they're not documented in `docker/README.md`), I expect users to copy an paste the instructions into their own stacks or just use them for trying things out. You can start a vanilla R process with the same container at the cost of image size. I added that option to the docs.

An alternative would be to provide two (or more) images, which differ in the base image. Then the question is: How many versions of `r-ver` ? Would you regularly update the different version tags and update the configurations? Is it worth the effort to provide a `lightgbmr/r-ver:x.x.x` for many different versions?

This only makes sense for me if LightGBM would publish "official" automated builds, and I wouldn't expect you to do that. That's why I went with the "training" option as you put it.

Reply:
Makes sense to me, thank you for the explanation. I really like the clarification you added (e.g. saying "...so that you can develop your analysis using LightGBM or simply try out the R package")!